fastlane/Fastfile

```diff
@@ -0,0 +1,242 @@
+# frozen_string_literal: true
+
+require './Constants/Constants'
+require './Managers/VersioningManager'
+require './Managers/BuildManager'
+require './Managers/SymbolManager'
+require './Managers/DistributionManager'
+
+versioning_manager = VersioningManager.new(
+  fastlane: self,
+  project_path: Constants.PROJECT_PATH,
+  main_target_name: Constants.MAIN_TARGET_NAME
+)
+
+builder = BuildManager.new(
+  fastlane: self,
+  scheme_name_staging: Constants.SCHEME_NAME_STAGING,
+  scheme_name_production: Constants.SCHEME_NAME_PRODUCTION,
+  scheme_name_appstore: Constants.SCHEME_NAME_APP_STORE,
+  bundle_identifier_staging: Constants.BUNDLE_ID_STAGING,
+  product_name_staging: Constants.PRODUCT_NAME_STAGING,
+  product_name_staging_test_flight: Constants.PRODUCT_NAME_STAGING_TEST_FLIGHT,
+  product_name_production: Constants.PRODUCT_NAME_PRODUCTION
+)
```

Comment:
From the latest [document](https://www.notion.so/nimblehq/Project-Configurations-e66e3e19ecc541028c605481b28b1bd0). I think we need to check and reduce the parameter here.

One more thing, instead of putting every parameter in the init, let's move it to function for easy to expand later, for example
```rb
builder = BuildManager.new(fastlane: self)

builder.build_staging_for_firebase(
  scheme_name: Constants.SCHEME_NAME_STAGING, 
  product_name: Constants.PRODUCT_NAME_STAGING
)
```

Reply:
What do you think?

Reply:
Good idea.